dali/test/python/test_pipeline.py

```diff
@@ -1581,10 +1581,9 @@ def test_pipeline_wrong_device_id():
     pipe = dali.Pipeline(batch_size=1, num_threads=1, device_id=-123)
     with pipe:
         pipe.set_outputs(np.int32([1,2,3]))
-    with assert_raises(RuntimeError) as x:
+    with assert_raises(RuntimeError, regex="device_id"):
```

Comment:
Do we really expect to have only `device_id` in the error message? I would extend the regexp here,

Reply:
I can take a look what the message looks like. I don't remember what it was - but judging by the fact that I've only left `device_id`, it wasn't very clear.

Reply:
So I would fix the message at the first place.